dms/controllers/portal.py

```diff
@@ -24,42 +36,44 @@ def _dms_check_access(self, model, res_id, access_token=None):
     def _prepare_home_portal_values(self, counters):
         values = super()._prepare_home_portal_values(counters)
         if "dms_directory_count" in counters:
-            ids = request.env["dms.directory"]._get_own_root_directories()
-            values["dms_directory_count"] = len(ids)
```

Comment:
Please don´t remove the `_get_own_root_directories()` method, it is necessary.

Example use case:
- Directory A
-- Subdirectory A1
-- Subdirectory A2

Depending on the user's permissions, he may only have access to the following directories:
- Subdirectory A1
- Subdirectory A2

and those directories should appear as “_roots_”, for example in portal view.